deploy/cluster-operator/templates/crds.yaml

```diff
@@ -1,6 +1,49 @@
-{{- if .Values.crds.create }}
-{{- range $path, $bytes := .Files.Glob "crds/*.yaml" }}
-{{ $.Files.Get $path }}
----
-{{- end }}
-{{- end }}
```

Comment:
This is the recommended way to install CRDs. It will be required when we switch to helm3. Is there a reason it's being checked into the templates folder?